packages/grafana-data/src/types/config.ts

```diff
@@ -238,6 +238,8 @@ export interface GrafanaConfig {
   reportingStaticContext?: Record<string, string>;
   exploreDefaultTimeOffset?: string;
   exploreHideLogsDownload?: boolean;
+  /** Custom quick ranges for the time picker */
+  quickRanges?: Array<{ from: string; to?: string; display: string }>;
```

Comment:
Can we use `TimeOption` instead of `{ from: string; to?: string; display: string }`?